docker/Dockerfile

```diff
@@ -19,7 +19,7 @@
 ARG BASE_IMAGE=ubuntu:18.04
 
 FROM ${BASE_IMAGE} AS compile-image
-
+ARG BASE_IMAGE=ubuntu:18.04
```

Comment:
Looks to be duplicate?

Reply:
To refer the variable for installing cuda version specific packages, it needs to be reloaded after FROM statement.